src/components/SelectedAnnotation.jsx

```diff
@@ -23,10 +26,16 @@ class SelectedAnnotation extends React.Component {
     this.closePrompt = this.closePrompt.bind(this);
     this.deletePrompt = this.deletePrompt.bind(this);
     this.closePopup = this.closePopup.bind(this);
+    this.setModern = this.setModern.bind(this);
+    this.toggleScriptOptions = this.toggleScriptOptions.bind(this);
+    this.closeDropdown = this.closeDropdown.bind(this);
+    this.previousScript = this.previousScript.bind(this);
+    this.nextScript = this.nextScript.bind(this);
 
     this.state = {
       annotationText: '',
-      showKeyboard: true
+      showKeyboard: true,
```

Comment:
`showKeyboard` is in the state instead of the duck; this means that every time a user starts transcribing, the onscreen keyboard will always be open. (As opposed to, e.g. the `activeScript` variable, whose state is persistent.)

Wouldn't it be a better idea to move this to the duck so it remembers the user's choice?